isaac2/Assets/Scenes/SampleScene.unity

```diff
@@ -134,7 +217,7 @@ GameObject:
   - component: {fileID: 519420031}
   - component: {fileID: 519420029}
   - component: {fileID: 519420033}
-  m_Layer: 8
+  m_Layer: 0
```

Comment:
Looks like we lost the Player layer when you merged main back into your branch

Reply:
I re-added the player layer but I only applied to the parent gameobject rather than to children too. Hopefully thats right.

Reply:
Doesn't really matter either way rn so probably good just as long as its still on Bullet as well (I assume so since its not coming up as a diff)